src/main/java/com/todoist/pojo/User.kt

```diff
@@ -30,6 +30,7 @@ open class User<T : TzInfo, F : Features>(
     open var uniquePrefix: Long?,
     open var hasPassword: Boolean,
     open var verificationStatus: VerificationStatus,
+    open var multiFactorAuthEnabled: Boolean,
```

Comment:
If `null` is sent for this value from the backend, I believe Jackson will translate it to `false`?